Roslyakov Evgeny/index.html

```diff
@@ -0,0 +1,73 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href='//fonts.googleapis.com/css?family=Raleway:100,300,400,600' rel='stylesheet' type='text/css'>
+    <link href="https://unpkg.com/ionicons@4.5.10-0/dist/css/ionicons.min.css" rel="stylesheet">
+    <link type="text/css" rel="stylesheet" href="style.css">
+    <script src="main.js" defer></script>
+</head>
+<body>
+    <div class="main-container">
+        <header class="task-menu">
+            <nav class="navigation">
+                <span>task sort</span>
+                <a class="button sort-done">checked</a>
+                <a class="button sort-undone">pending</a>
+                <span>by deadline</span>
+                <a class="button sort-tomorrow">tomorrow</a>
+                <a class="button sort-week">week</a>
+                <a class="button button-primary add-new-task">add new task</a>
+            </nav>
+        </header>
+        <section class="todo-list">
+            <div class="tasks-container">
+                <!--<section class="task task-checked" id="idx-%ID%">
+                    <nav class="task__edit">
+                        <button class="ion-button task-check"><i class="ion-icon ion-md-checkmark"></i></button>
+                        <button class="ion-button task-delete"><i class="ion-icon ion-md-close"></i></button>
+                    </nav>
+                    <div class="task__content">
+                        <h6 class="task__title">%TITLE%</h6>
+                        <div class="task__description"><span>%DESCRIPTION%</span></div>
+                    </div>
+                    <div class="task__meta-info">
+                        <span class="task__date">date: %DATE%</span>
+                        <span class="task__deadline">deadline: %DEADLINE%</span>
+                    </div>
+                </section>-->
+            </div>
+            <section class="new-task">
+                <form class="new-task__form">
+                    <h6 class="new-task__header-wrapper">
+                        <span class="new-task__header">new task</span>
+                        <i class="ion-icon ion-md-close form-close"></i>
+                    </h6>
+                    <input type="text" class="new-task__title" placeholder="task title">
+                    <textarea class="new-task__description" rows="8" placeholder="task description"></textarea>
+                    <div class="form-footer">
+                        <input type="button" class="button button-primary save-button" value="save">
+                        <div class="deadline-radios">
+                            <span>deadline:</span>
+                            <ul class="radio-wrapper">
+                                <li>
+                                    <label><input type="radio" class="new-task__deadline" name="set-deadline" value="none">none</label>
```

Comment:
I so that this value is set as default in js part, what do you think about putting here checked attribute to show the user that it is a default?